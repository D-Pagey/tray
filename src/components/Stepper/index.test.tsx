import React from 'react';
import { render } from '@testing-library/react';
import { Stepper } from '.';

describe('Stepper component', () => {
    it('should render', () => {
        const { container } = render(<Stepper />);
        expect(container.firstChild).toMatchSnapshot();
    });

    it.todo('should progress the step - have to fill in the form');
});
