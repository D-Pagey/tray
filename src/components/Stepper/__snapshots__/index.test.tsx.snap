// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Stepper component should render 1`] = `
<div>
  <h2
    class="sc-fzokOt eemgQE"
  >
    Sign Up Form
  </h2>
  <div
    class="sc-fzqBZW cZzLuM"
  >
    <button
      class="sc-fzqNJr kKQZEm"
      type="button"
    >
      User
    </button>
    <button
      class="sc-fzqNJr jCBTEO"
      type="button"
    >
      Privacy
    </button>
    <button
      class="sc-fzqNJr jCBTEO"
      type="button"
    >
      Done
    </button>
  </div>
  <div
    class="sc-AxhCb iLrMEl"
  >
    <p
      class="sc-AxhUy hQDqcJ"
    >
      <span
        class="sc-AxgMl lYEcR"
      >
        *
      </span>
       indicates a required field
    </p>
    <form
      action="#"
      class="sc-AxheI fFzjlp"
    >
      <label
        class="sc-AxmLO dkqJhP"
        for="name"
      >
        Name: 
        <span
          class="sc-AxgMl lYEcR"
        >
          *
        </span>
      </label>
      <input
        class="sc-Axmtr lkhMLB"
        data-testid="signUpNameInput"
        id="name"
        name="name"
        placeholder="Joe Bloggs"
        type="text"
        value=""
      />
      <label
        class="sc-AxmLO dkqJhP"
        for="role"
      >
        Role:
      </label>
      <input
        class="sc-Axmtr lkhMLB"
        data-testid="signUpRoleInput"
        id="role"
        name="role"
        placeholder="Software Engineer"
        type="text"
        value=""
      />
      <label
        class="sc-AxmLO dkqJhP"
        for="email"
      >
        Email: 
        <span
          class="sc-AxgMl lYEcR"
        >
          *
        </span>
      </label>
      <input
        class="sc-Axmtr lkhMLB"
        data-testid="signUpEmailInput"
        id="email"
        name="email"
        placeholder="hello@email.com"
        type="email"
        value=""
      />
      <label
        class="sc-AxmLO dkqJhP"
        for="password"
      >
        Password: 
        <span
          class="sc-AxgMl lYEcR"
        >
          *
        </span>
      </label>
      <input
        class="sc-Axmtr lkhMLB"
        data-testid="signUpPasswordInput"
        id="password"
        name="password"
        placeholder="Secret password..."
        type="password"
        value=""
      />
      <button
        class="sc-AxirZ esfovg"
        data-testid="signUpSubmitButton"
        type="submit"
      >
        Submit
      </button>
    </form>
  </div>
</div>
`;
